cmds/subvolume.c

```diff
@@ -179,19 +167,6 @@ static int create_one_subvolume(const char *dst, struct btrfs_qgroup_inherit *in
 	}
 	dstdir = path_dirname(dupdir);
 
-	if (!test_issubvolname(newname)) {
```

Comment:
This check should be preserved, either here or moved to libbtrfsutil. It checks the name for special characters that are either reserved '..' or not allowed in file name '/'. Kernel verifies that as well but generall in progs we do all validation if possible.